core/primitives/src/views.rs

```diff
@@ -127,13 +127,13 @@ impl From<AccountView> for Account {
 
 impl From<ContractCode> for ContractCodeView {
     fn from(contract_code: ContractCode) -> Self {
-        ContractCodeView { code: contract_code.code, hash: contract_code.hash }
+        ContractCodeView { code: contract_code.code().to_vec(), hash: contract_code.hash().clone() }
```

Comment:
looks like there is some unnecessary clone given that we take ownership of contract_code here.

Reply:
hmm in fact I think all the `contract_code.hash().clone()` calls could be replaced with `*contract_code.hash()` since `CryptoHash` implements the `Copy` trait. Am I right or you'd prefer explicit cloning?

Reply:
I think since it's Copy, `*contract_code.hash()` is fine since don't clone. Bowen probably means contract_code is `ContractCode` instead of `&ContractCode` so that there should not need a clone.

Reply:
got it, fixed